coinoffers.py

```diff
@@ -132,8 +134,12 @@ def bittilasku():
 
 
 def main():
-    unsorted_offers = bitcoinde() + localbitcoins() + coinmotion()\
-    # + bittilasku()  # not a seller, but a bill paying service
+    if coinmotion():
```

Comment:
Instead of special casing Coinmotion here, I would
- collect the list of offers or an error status from each of the download/decoder functions,
- merge the offers into the unsorted list,
- merge the errors into a separate data structure, and
- modify the JSON structure to have separate sections for offers and errors.

I guess it's a matter of taste whether to pass errors in the return value or an exception object.

What do you think?